cmd/kubeadm/app/phases/addons/proxy/proxy.go

```diff
@@ -118,6 +118,12 @@ func createKubeProxyConfigMap(cfg *kubeadmapi.ClusterConfiguration, localEndpoin
 		return errors.Wrap(err, "error when parsing kube-proxy configmap template")
 	}
 
+	if printManifest {
+		fmt.Printf("%s\n", configMapBytes)
+		fmt.Println("---")
```

Comment:
```suggestion
		fmt.Println("---")
		fmt.Printf("%s\n", configMapBytes)
```

we should print the deamonset here too?